src/main/java/uk/gov/companieshouse/model/dto/email/SignatoryToSignEmailData.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.companieshouse.model.dto.email;
+
+public class SignatoryToSignEmailData extends EmailData {
+
+    private String to;
+    private String subject;
```

Comment:
I've made this change as part of my story here, https://github.com/companieshouse/dissolution-api/pull/43/files#diff-ad031a1444c490741e9e363232d52893, but since `to` and `subject` will be included for all emails, this could be added to the `EmailData` model.

Reply:
Rather than us both making the change, are you happy to remove these properties from SignatoryToSignEmailData once your PR is ready for review? I'm expecting to merge this soon.